### DISCUSSION TITLE
Land Tax/Land Fee

### DISCUSSION THREAD

**Discussion by michalptacnik** [liberland/constitution#582] [Ideas]
_Taxation is theft, purely and simply even though it is theft on a grand and colossal scale which no acknowledged criminals could hope to match. It is a compulsory seizure of the property of the State’s inhabitants, or subjects._ - Murray N. Rothbard, the Ethics of Liberty.

The Land Tax represents a consensual instrument of voluntary taxation. Liberland sets this instrument up using an adhesive contract mandated by the constitution. That means an arrangement where no negotiation is permitted, "take it or leave it"; most supplier-consumer interactions are adhesive. Hence, this is not unusual on the market.
However, this is unusual in the Real Estate world and the world of Public-Private interactions because the governments of legacy states feel little need to keep their fees and taxes consensual; they legislate them and enforce them upon the people.

The main points of the Land Tax make it unlike normal taxation:

* there is no special tax bureau to enforce it;
* there are no privileged agents to do so;
* the Government will have to go to a Collection Agency;
* the proceedings from the sales of the property in question will mainly go to the owner, the Government; and
only keeping the portion, the owner must pay in Land Tax.

I want to open the discussion about this institute here. The proposed motions are:
1. Further making this more standard by polishing "the rough edges";
turning this into a "real property tax" whilst abolishing our uncompromising stance against involuntary coercive taxation;
2. Abolishing the Land Tax whilst drafting in the Real Estate Law a "System Singapore." - in that legal world, Liberland is mandated by the constitution never to sell land, instead renting it;
3. Abolish the Land Tax without replacement, losing it as a revenue or land control method.

**Comment by Claudia1207-pixel** [comment]
1.  I agree this motion would be against involuntary coercive taxation.
2. This would be sort of a land tax, though more just in a way because everyone could negotiate the rent amount, I suppose. 
3. A revenue would be gained by means of voluntary contributions, which can be negotiated as everything else by the market. For example, the government could ask the citizens to help with some tasks, the cost of something, etcetera, and everyone can choose with what and how much they want to contribute.  On the other hand,  I think a land control method would be unnecessary if people using the land follow some guidelines as to what is best for the future of LL. For example, is there a need for more than a couple of marinas? And that would be settled by the market and could be negotiated in an agreement, also, if needed, according to the piece of land´s location, for example. 

In conclusion, I support the third motion.

**Reply by Claudia1207-pixel** [reply]
Sorry! 
Number 1 should read "it would be coercive taxation."

**Reply by michalptacnik** [reply]
Thank you! Options 2 and 3 are very similar: they purge this institute from the Constitution, whereas two allow it to be replaced by what is effectively a market-based rent. We should, however, make sure that this rent is set up by law and equally (not necessarily "the same") for all, based on some mathematical formulas and other systemic considerations. E.g. "purpose of the property" gives the initial number to X, and then consideration is given to some degree to the real estate market price.
The problem with option 3 is that it would remove Liberland's ability to do zoning and impair its ability to get land to be used by those who hold it for speculation. Given the tiny size of Liberland, this could be fatal and could also be done maliciously to harm us directly. Zoning is also necessary (light zoning); otherwise, we would have no direct way to prevent en bloc ideas like "I will store nuclear waste in the middle of the main residential quarter of Liberland." Of course, the judge-made-law might deal with it, in time, but when toxic waste or other hazards are concerned, time is in short supply.

**Comment by Claudia1207-pixel** [comment]
Hi, Michal!
About the zoning, at least what I´ve seen happen here in Guatemala is that
can get corrupted and whoever pays gets to decide what goes on their land,
regardless of the law.
As in any other matter, shouldn´t it be like everything goes, except if it
harms someone else, and then the courts would solve it?
Best regards,
Claudia

**Reply by michalptacnik** [reply]
That would be theoretically pure & ideal, but we are very time-bound. In a big State, sure, we can wait, but we need an orderly, explosive growth, especially in the beginning. I am afraid just to let this one (the use of land) "flow" - one badly placed landfill we have no way to evict administratively, and we are done for, even after the immense job of getting full access to the land. This is an attack vector for enemies too, not just for the natural human greed (which we of course don't try to fight). I would suggest a very simple zoning system, we don't have the money to maintain & enforce anything more. Or then, Singapore, which would not be zoning at all but just the proprietor, Liberland, doing what they want with their property. "I want to hire this land to build a landfill with nuclear waste depo" - ""sorry, but we don't intend for this land parcel to be used in this manner."

**Reply by Claudia1207-pixel** [reply]
Oh, I understand now. 
Therefore, I think your proposal for Singapore-style management land would be best.
This arrangement could change sometime in the future, I suppose?

**Reply by michalptacnik** [reply]
It certainly could, it would be enshrined in Law, not in the Constitution itself. How this would look: there would be no Land Tax in the Constitution, but it would be entirely enshrined in the Real Estate Law (the draft of which you can see [here](https://github.com/liberland/laws/issues/24)). There, the Initial Land Ownership clause would change in that the whole Gornja Siga shall be owned by Liberland and Liberland shall keep it "for indeterminate time". If we should want to change this later on, we will have to amend a "mere" Law, rather than the Constitution.

**Reply by jonathancross** [reply]
"Singapore style" sounds good as a start to me as well.
